cmd/run.go

```diff
@@ -910,6 +911,11 @@ var runCmd = &cobra.Command{
 
 		span.End()
 
+		raftNode, originalErr := raft.NewRaftNode(logger, conf.Global.Raft)
+		if originalErr != nil {
```

Comment:
Isn't this a Fatal error? If we can't configure and create a raft node, provided that is configured, we must stop (I guess).

Reply:
Nice catch!, Addressed in [61945e3](https://github.com/gatewayd-io/gatewayd/pull/636/commits/61945e3b10027fe05b2da893841453cdfb07125c)